target/linux/mediatek/image/mt7623.mk

```diff
@@ -1,53 +1,95 @@
 KERNEL_LOADADDR := 0x80008000
+DEVICE_VARS += UBOOT_TARGET UBOOT_OFFSET UBOOT_ENVSIZE
 
-ifneq ($(CONFIG_BANANA_PI_BOOT_PARTSIZE),)
+ifneq ($(CONFIG_MTK_BOOT_PARTSIZE),)
```

Comment:
Most other target add "SD" here:
CONFIG_AXS10X_SD_BOOT_PARTSIZE
CONFIG_OMAP_SD_BOOT_PARTSIZE
AT91_SD_BOOT_PARTSIZE
CONFIG_SUNXI_SD_BOOT_PARTSIZE

In contrast, I wonder whether the target name needs to be included here, as it's defined per target anyway. Adding the device name would make sense if there are different values for different targets, but if it's the same across the target the target name won't provide any information.

Reply:
I think it had SD in it once, but it covers eMMC too so I figured that was misleading.

Reply:
err, you are right, I'm too much consumer there, for me it's all "SD" already ...